test/resources/test_survey.py

```diff
@@ -0,0 +1,55 @@
+import json
+import unittest
+
+import requests_mock
+
+from ras_backstage import app
+
+
+url_get_survey_list = '{}{}'.format(app.config['RM_SURVEY_SERVICE'], 'surveys')
+with open('test/test_data/survey/survey_list.json') as json_data:
+    survey_list = json.load(json_data)
+
+
+class TestSurvey(unittest.TestCase):
+
+    def setUp(self):
+        self.app = app.test_client()
+
+    @requests_mock.mock()
+    def test_get_survey_list(self, mock_request):
+        mock_request.get(url_get_survey_list, json=survey_list)
+
+        response = self.app.get('/backstage-api/v1/survey/surveys')
+
+        self.assertEqual(response.status_code, 200)
+        self.assertIn('BRES'.encode(), response.data)
+        self.assertIn('BRUS'.encode(), response.data)
+
+    @requests_mock.mock()
+    def test_get_survey_list_no_surveys(self, mock_request):
+        mock_request.get(url_get_survey_list, status_code=204)
+
+        response = self.app.get('/backstage-api/v1/survey/surveys')
+
+        self.assertEqual(response.status_code, 200)
```

Comment:
Seems a little inconsistent to return 204 from one service with no data to the return 200. Seems like for consistency we should probably do one or the other?

Reply:
I do kind of agree but doing it this way does simplify the logic in frontstage.
Just returning an empty array to frontstage is nicer than having to check for 200 and 204.

We don't use 204 in any other services I don't think (for some we return 404) so it might be nice to change that to 200 in the survey service for consistency.

Reply:
One of the reasons for this backstage service was to move logic away from the frontend, this does that